src/lib.rs

```diff
@@ -1,4 +1,22 @@
 #![forbid(unsafe_code)]
+#![warn(
```

Comment:
I like that you are bringing this up! I briefly thought about that.

It’s probably not even needed to bump the MSRV as they should be ignored on older versions where they are still unknown? Lints will be checked on stable and for older versions unknown-lints should be ignored anyway.

Also it would affect examples then too I think?

Reply:
Our MSRV policy is 2 versions behind stable - mainly because that matches policies of our deps - so 1.74.0 is fine here. Can you take a look at whether this can be set without a tonne of work on the examples - if it can, then let's do it there.

Reply:
I'm going to mark this one as resolved - we can do this as a secondary step if needed.

Reply:
I waited with this until #976 is merged :innocent:

Reply:
Done :sparkles: